app.js

```diff
@@ -0,0 +1,53 @@
+const addBook = document.querySelector('.book-form');
+const bookShelf = document.querySelector('.items');
+
+const store = localStorage.getItem('bookCollection');
+const bookCollection = JSON.parse(store) || [];
+
+const renderBooks = () => {
+  bookShelf.innerHTML = '';
+
+  bookCollection.forEach((book, index) => {
+    const html = `
+      <article class="book-list">
+        <ul class="book-details">
+          <li class="title-list">
+            <span>${book.title}</span>
+          </li>
+          <li class="author-list">
+            <span>${book.author}</span>
+          </li>
+        </ul>
+        <button type="submit" class="remove" data-index="${index}">Remove</button>
+      </article>
+    `;
+
+    bookShelf.innerHTML += html;
+  });
+};
+
+renderBooks();
+
+addBook.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  const bookTitle = addBook.title.value.trim();
+  const bookAuthor = addBook.author.value.trim();
+
+  if (bookTitle.length && bookAuthor.length) {
+    bookCollection.push({ title: bookTitle, author: bookAuthor });
+    localStorage.setItem('bookCollection', JSON.stringify(bookCollection));
+    renderBooks();
+
```

Comment:
- [ ] Kindly consider changing this into a function that add a new book to the collection, then you can use the new function inside the event listner